ERCS/erc-0000.md

```diff
@@ -0,0 +1,68 @@
+---
+eip: 0000
+title: Multi-Owner Non-Fungible Tokens (MO-NFT)
+description: A new type of non-fungible token that supports multiple owners, allowing shared ownership and transferability among users.
+author: James Savechives (@jamesavechives) <james.walstonn@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/multi-owner-nfts-discussion-thread/12345
```

Comment:
Please create a topic in Eth Magicians with a link to this PR.

Reply:
OK, sure

Reply:
Hello, when I try to post the topic in Eth Magicians, it alwasy returns such error : "An error occurred: You are not permitted to view the requested resource."

What I want to post is :

Title :  Discussion on EIP-7743: Multi-Owner Non-Fungible Tokens (MO-NFT)

Body :

Hi everyone,

I’d like to start a discussion on my new EIP-7743: Multi-Owner Non-Fungible Tokens (MO-NFT).

Abstract:
This EIP proposes a new standard for non-fungible tokens (NFTs) that supports multiple owners. The MO-NFT standard allows a single NFT to have multiple owners, reflecting the shared and distributable nature of digital assets. This model also incorporates a mechanism for value depreciation as the number of owners increases, maintaining the principle that less ownership translates to more value.

Motivation:
Traditional NFTs enforce a single-ownership model, which does not align with the inherent duplicability of digital assets. MO-NFTs allow for shared ownership, promoting wider distribution and collaboration while maintaining secure access control. This model supports the principle that some valued information is more valuable if fewer people know it, hence less ownership means higher value.

You can view the full proposal and PR [here](https://github.com/ethereum/ERCs/pull/539).

I welcome any feedback, suggestions, or discussions on this proposal.

Thank you!

Reply:
> Please create a topic in Eth Magicians with a link to this PR.

Done!